packages/datagrid/src/components/DatasetSerializer/datasetSerializer.js

```diff
@@ -55,9 +56,26 @@ export function sanitizeAvro(avro) {
  * @param {boolean} optional;
  */
 export function getTypeValue(type, optional) {
+	if (typeof type === 'string') {
+		return `${type}${optional ? '' : '*'}`;
+	}
 	return `${type.dqType || type.type}${optional ? '' : '*'}`;
 }
 
+/**
+ * isNull - check if the type is null
+ *
+ * @param  {type} string|object type to check
+ * @return {boolean} return if the type is null
+ */
+export function isNull(type) {
```

Comment:
Why do you export it ?
It's supposed to be internal. If it's for the tests, this is useless, don't export internal things for tests, they are tested with the exposed api implicitly.
Doing that makes it less maintainable, as you can change the implementation by refactoring internal code, it will break the tests even if there is no impact on exposed api.

Reply:
done remove it